ayup/portfolio/src/main/webapp/script.js

```diff
@@ -25,14 +25,16 @@ function addRandomFact() {
     "🐕 I have three dogs!"];
   
   // Pick a random fact.
-  var fact = facts[Math.floor(Math.random() * facts.length)];
+  var newFact = facts[Math.floor(Math.random() * facts.length)];
 
-  // Add it to the page.
-  const factContainer = document.getElementById('fact-container');
+  // Get the current fact
+  const factContainer = document.getElementById('js-fact-container');
+  currentFact = factContainer.innerHTML;
```

Comment:
`const currentFact = ...` to make this a local variable. Otherwise this adds a property `currentFact` to `window`.

Reply:
Done